reviews component to antd timeline transfare and bug fix

costashush opened the pull request:
image white space elimination
reviews to antd timeline transfare

andreykishtov commented on src/components/Reviews/Review.js:
```diff
@@ -2,44 +2,31 @@ import React from 'react';
+  padding: 20px;
   border-radius: 10px;
-  margin-left: 100px;
+  margin-left: 10px;
```
if you create more then one margin direction use margin 2em 10px 0 0;

costashush replied:
You rigth I will fix it

costashush commented:
<img width="1763" alt="2017-11-26" src="https://user-images.githubusercontent.com/26918005/33240404-013a615c-d2be-11e7-876d-36ba029aad8b.png">

costashush commented:
Ok will fix it

costashush commented:
fixed the image show component (padding alteration)
added scroll to  reviews
<img width="1763" alt="2017-11-27" src="https://user-images.githubusercontent.com/26918005/33263867-1bc6c82e-d374-11e7-9268-9579997a241a.png">
